spec/lib/percy/percy_spec.rb

```diff
@@ -39,16 +40,20 @@
         visit 'http://example.com'
         Percy.snapshot(page, { name: 'minHeight', minHeight: 2000 })
       end
+      it 'recognizes enableJavascript' do
+        visit 'http://example.com'
```

Comment:
This won't actually show anything different with JS on or off. In a follow up PR let's move to testing with https://github.com/percy/sdk-test-website and we can add JS detection to that site.

Reply:
Yeah, that's the TODO in my PR description. Can you give the SDK team access to that repo? It 404s for me at the moment. 

I'm thinking we're going to need to add something like canvas since that isn't captured by our SDKs yet and _requires_ JS to build the element.